src/App.js

```diff
@@ -0,0 +1,84 @@
+import './App.css';
+import { nanoid } from 'nanoid';
+import { useState, useEffect } from 'react';
+import List from './components/List';
+
+const getLocalStorage = () => {
+  let list = localStorage.getItem("list")
+  if (list) {
+    return (list = JSON.parse(localStorage.getItem("list")))
```

Comment:
You can refactor the code to
```suggestion
    return JSON.parse(list)
```